pypeit/wavetilts.py

```diff
@@ -147,6 +170,75 @@ def spatid_to_zero(self, spat_id):
         mtch = self.spat_id == spat_id
         return np.where(mtch)[0][0]
 
+    def show(self, waveimg=None, wcs_match=True, in_ginga=True, show_traces=False):
+        """
+        Show in ginga or mpl Tiltimg with the tilts traced and fitted overlaid
+
+        Args:
+            waveimg (`numpy.ndarray`_, optional):
+                Image with the wavelength solution.
+            wcs_match (bool, optional):
+                If True, use this image as a reference image for the WCS and match all
+                image in other channels to it.
+            in_ginga (bool, optional):
+                If True, show the image in ginga. Otherwise, use matplotlib.
+            show_traces (bool, optional):
+                If True, show the traces of the tilts on the image.
+
+        """
+        # get slits
+        if (Path(self.calib_dir).resolve() / self.slits_filename).exists():
+            slits = slittrace.SlitTraceSet.from_file(Path(self.calib_dir).resolve() / self.slits_filename)
+            slitmask = slits.slit_img(initial=True, flexure=self.spat_flexure)
+            left, right, mask = slits.select_edges(flexure=self.spat_flexure)
+            gpm = mask == 0
+        else:
+            slits = None
+            msgs.warn('Could not load slits to show with tilts image.')
+
+        # get tiltimg
+        if (Path(self.calib_dir).resolve() / self.tiltimg_filename).exists():
+            tilt_img_dict = buildimage.TiltImage.from_file(Path(self.calib_dir).resolve() / self.tiltimg_filename)
+            tilt_img = tilt_img_dict.image * (slitmask > -1) if slits is not None else tilt_img_dict.image
+        else:
+            msgs.err('Tiltimg not found.')
+
+        # get waveimg
+        if waveimg is None and in_ginga:
+            wv_calib_name = wavecalib.WaveCalib.construct_file_name(self.calib_key, calib_dir=self.calib_dir)
+            if Path(wv_calib_name).resolve().exists() and slits is not None:
+                wv_calib = wavecalib.WaveCalib.from_file(wv_calib_name)
+                tilts = self.fit2tiltimg(slitmask, flexure=self.spat_flexure)
```

Comment:
This step (`fit2tiltimg`) can take a while... I wonder if it's worthwhile putting in `msgs.info("Converting tilt fit to an image")` or perhaps informing the user in `self.fit2tiltimg` when the tilt is being evaluated for each slit, just so it doesn't look like it's hanging?

Reply:
Added a `msgs.info` in `self.fit2tiltimg`